modules/ModuleBuilder/MB/MBLanguage.php

```diff
@@ -119,6 +119,11 @@ public function generateModStrings()
     public function getModStrings($language='en_us')
     {
         $language .= '.lang.php';
+
+        if(isset($this->strings['_override_'. $language])){
+            $this->strings['en_us.lang.php'] = array_replace($this->strings[$language],$this->strings['_override_'. $language]);
```

Comment:
Hi @JackBuchanan, why are we setting this in `$this->strings['en_us.lang.php']` and not in `$this->strings[$language]`

Reply:
Good spot @clemente-raposo updated the code with your suggestion